packages/application-generic/src/services/in-memory-provider/in-memory-provider.service.spec.ts

```diff
@@ -147,8 +146,6 @@ describe('In-memory Provider Service', () => {
       });
 
       it('should instantiate the provider properly', async () => {
-        expect(inMemoryProviderService.isClusterMode()).toEqual(true);
```

Comment:
Superfluous check as we inject the cluster mode in the InMemoryProviderService now when setting the tests.